tests/test_sql_magic.py

```diff
@@ -90,6 +48,7 @@ def test_does_not_assign_exceptions_to_variable(self, sql_magic, ipython_shell):
     def test_unknown_datasource_handle_produces_expected_exception(self, sql_magic, capsys):
         from noteable_magics.sql.connection import Connection, UnknownConnectionError
 
```

Comment:
We don't clear out the connections every test anymore, in that the CRDB connection is session-scoped. So this test needs to be adjusted an iota.